projects/media-viewer/src/lib/viewers/image-viewer/image-viewer.component.html

```diff
@@ -3,7 +3,15 @@
     <mv-annotation [annotation]="annotation" [zoom]="zoom"
                    (update)="updateAnnotation($event)"></mv-annotation>
   </ng-container>
-  <div *ngIf="!errorMessage" id="image-container" class="image-container" [ngStyle]="{ transform: zoomStyle }">
+  <div #newRectangle class="newRectangle" (mousemove)="onMouseMove($event)" (mouseup)="onMouseUp($event)"></div>
```

Comment:
should this only be there in drawMode?

Reply:
it's display none by default

Reply:
but the div is still in the DOM right? so the event listeners are probably also added

Reply:
okay added if